Packs/Threatgraph/Integrations/Threatgraph/Threatgraph.yml

```diff
@@ -0,0 +1,129 @@
+category: Data Enrichment & Threat Intelligence
+commonfields:
+  id: Threatgraph
+  version: -1
+configuration:
+- defaultvalue: https://falconapi.us-2.crowdstrike.com
+  display: Server URL (e.g. https://falconapi.us-2.crowdstrike.com)
+  name: url
+  required: true
+  type: 0
+- display: Fetch incidents
+  name: isFetch
+  required: false
+  type: 8
+- additionalinfo: Threatgraph API key
+  display: API Key
+  name: apikey
+  required: true
+  type: 4
+- additionalinfo: Threatgraph API secret
+  display: API Secret
+  name: apisecret
+  required: true
+  type: 4
+- defaultvalue: "1"
+  display: Incidents Fetch Interval
+  name: incidentFetchInterval
+  required: false
+  type: 19
+- display: Fetch indicators
+  name: feed
+  required: false
+  type: 8
+- additionalinfo: Indicators from this integration instance will be marked with this
+    verdict
+  display: Indicator Verdict
+  name: feedReputation
+  options:
+  - Unknown
+  - None
+  - Benign
+  - Good
+  - Suspicious
+  - Malicious
+  - Bad
+  required: false
+  type: 18
+- additionalinfo: Reliability of the source providing the intelligence data
+  defaultvalue: F - Reliability cannot be judged
+  display: Source Reliability
+  name: feedReliability
+  options:
+  - A - Completely reliable
+  - B - Usually reliable
+  - C - Fairly reliable
+  - D - Not usually reliable
+  - E - Unreliable
+  - F - Reliability cannot be judged
+  required: true
+  type: 15
+- defaultvalue: indicatorType
+  display: ""
+  name: feedExpirationPolicy
+  options:
+  - never
+  - interval
+  - indicatorType
+  - suddenDeath
+  required: false
+  type: 17
+- defaultvalue: "20160"
+  display: ""
+  name: feedExpirationInterval
+  required: false
+  type: 1
+- defaultvalue: "240"
+  display: Feed Fetch Interval
+  name: feedFetchInterval
+  required: false
+  type: 19
+- additionalinfo: When selected, the exclusion list is ignored for indicators from
+    this feed. This means that if an indicator from this feed is on the exclusion
+    list, the indicator might still be added to the system.
+  display: Bypass exclusion list
+  name: feedBypassExclusionList
+  required: false
+  type: 8
+contentitemexportablefields:
+  contentitemfields:
+    definitionid: ""
+    fromServerVersion: ""
+    itemVersion: ""
+    packID: 0c6a24d3-1a25-4f55-83ad-357fad4fab7c
+    propagationLabels:
+    - edr-crowdstrike
+    - cloud-zscaler
+    toServerVersion: ""
```

Comment:
This looks unnecessary, please remove.

Reply:
We have not modified any config directly for this code, we used the XSOAR Marketplace front end only.